GOVERNMENT.md

```diff
@@ -0,0 +1,56 @@
+- Simple derivative signals and post-processed data
+- Subject information and simple experimental metadata
+
+### SNIRF Government
+The mission of the SNIRF government is to ensure the format's longevity and to encourage its adoption. Critical to this mission is the ability of the fNIRS community to expand or correct the specification as new developments emerge without creating issues and conflicts for users. The SNIRF government is responsible for moderating this process, and for collecting and disseminating important information to SNIRF users.
+
+## Bylaws
+
+## Article I. Steering Committee
+
+**Section 1.** Steering Committee Definition. The SNIRF Steering Committee is ultimately responsible for guiding the development of SNIRF in accordance with its mission. The committee must unanimously approve the releases of official and final changes to the specification, as well as changes to the SNIRF government or mission statement.
+
+**Section 2.** Election. Steering committee members must be nominated or self-nominated in advance of the end of a sitting member's term. The departing member is responsible for the organization of the election, and cannot be elected to remain in the same seat. All SNIRF Contributors, Maintainers, and Steering Committee members are elligble to vote in the election.
+
+**Section 3.** Terms. The Steering Committee is composed of between 3 and 5 members of the fNIRS community who are elected to serve 3 year terms.
+
+**Section 4.** Meetings. A majority of Steering Committee members must meet at least biannually. At least one Maintainer must be present at these meetings. These meetings should review the following:
+- The most recent Maintainers Report
+- Any pending releases or critical changes
+- project priorities and progress towards goals
+- Delegation of resources (i.e maintainers' attention) towards completion of goals
+
+The meeting should result in the authorship of a brief report.
+
+**Section 5.** Steering Committee Reports. The Steering Committee must author a brief report corresponding following each of its required biannaual meetings. This report must document the actions and considerations of the committee, and must be archived and disseminated via the GitHub repository. The report must include the names of the Steering Committee's members.
+
+**Section 6.** GitHub Repository Role. Steering Committee members must have the role of `Admin` on the GitHub repository.  
+
+**Section 7.** Specification Releases. The Steering Committee must unanimously approve of official releases and publish them via GitHub. Releases must be thoroughly documented and properly formatted as described in RELEASE.md. The Committee is subsequently responsible for alerting the community to the release.
+
+**Section 8.** Maintainer Election. Any member of the Steering Committee may at any time approve the election of a Maintainer. The Steering Committee member must give the Maintainer the role of `Maintain`.
```

Comment:
what do you mean by "approve the election of a Maintainer."? One the one hand it sounds like any steering committee member can add a maintainer at any time. But on the other hand, what you say is they can approve an election of a maintainer. I think you mean the former, but maybe you mean the latter. If you mean the latter, I guess this means a steering committee member at anytime to start an election with one nominee that goes out to the email list of contributors to then vote for that one nominee to become a maintainer. Is that what you mean?

Reply:
I meant that they can onboard one at any time. I will revise for clarity